### PR TITLE
Update copy

### PR DESCRIPTION
**Pull request by vonbearshark**
## Background
Text edits in the tickets. These are for the main ece reporter app copy changes, in `Home`, `Upload`, and `Roster`. Most changes relate to post-submission copy. 

Corresponding PR for help site copy changes in [that ece-reporter-support Jekyll site](https://github.com/ctoec/ece-reporter-support/pull/5)

## GitHub Issue
#1330
#1331
#1329 (partial)
#1353

## Validation Plan
Someone should double-check the UX for submission to make sure it's still 👍🏻 . We should also have @MorganGriffin-BloomWorks 👀 after it's deployed to QA

## Automated Testing
- [x] All unit tests are passing.
- [x] All e2e tests are passing.

### CODE REVIEW

**Review by JordanGuinn — APPROVED** [review]

**Review by bamader — APPROVED** [review]
Love me some text change approvals.

File: client/src/containers/Roster/Roster.tsx [hunk 1]
```diff
@@ -143,8 +128,40 @@ const Roster: React.FC = () => {
...
         <div className="grid-row flex-align-center">
           <div className="tablet:grid-col-9">
+            <p>
+              The Roster allows you to view and update the enrollment data you
+              submitted previously. Using the Roster, you can:
+            </p>
+            <ul className="margin-left-2 bx--list--unordered">
+              <li className="line-height-body-4 bx--list__item">
+                Review current enrollment, see enrollment data from past months,
+                and export data
```
Comment (bamader) [inline client/src/containers/Roster/Roster.tsx:141]: Think there's a typo here: It should say "Review current enrollments, ..." (plural) right?

**Review by bamader — APPROVED** [review]

**Comment by bamader** [comment]
Aside from a bug that's completely unrelated to any changes here, these copy changes are validated and proofed and ready to be merged!

**Review by jlhogan — APPROVED** [review]

**Comment by jlhogan** [comment]
@bamader the unrelated bug is causing e2e to fail? Is that bug live in prod? If yes, can we push this out ASAP? If not, can we hotfix this onto whatever is currently in prod and deloy that? @JordanGuinn

**Comment by bamader** [comment]
@jlhogan Bug is _not_ live in prod, thankfully (but I think is causing e2e to fail)! Jordan and I tracked it down yesterday and he's gonna get a fix up on it shortly (I would assume once it's not ungodly early in his time 😝 ).

**Comment by JordanGuinn** [comment]
That's correct - to close the gap on this, the bug we found is specific to our seeder process in test environments, so nothing that warrants an immediate prod hotfix.  Will be putting up a PR this morning, just cuz it's a really trivial update.

**Comment by JordanGuinn** [comment]
Also, looking at the browserstack video from this e2e test run, the failures look legitimate.  Gonna get a fix up before merging

**Comment by vonbearshark** [comment]
Catching up. Thanks, all! Good to merge now?

**Comment by JordanGuinn** [comment]
Not yet - testing e2e, as they legitimately failed last run.

**Comment by vonbearshark** [comment]
never a dull moment with copy changes...

**Comment by vonbearshark** [comment]
So this is all green, 👍🏻 , and ready to go. Do we want to block this for https://github.com/ctoec/data-collection/pull/1354? How do we want to handle the overlapping changes?

File: client/src/containers/TemplateDownload/TemplateDownload.tsx [hunk 2]
```diff
@@ -1,35 +1,21 @@
...
-import useSWR, { responseInterface } from 'swr';
 import { getH1RefForTitle } from '../../utils/getH1RefForTitle';
 import { CSVExcelDownloadButton } from '../../components/CSVExcelDownloadButton';
 import { ReactComponent as Image } from '../../images/SpreadsheetIllustration.svg';
 import { BackButton } from '../../components/BackButton';
-import { TemplateMetadataResponse } from '../../shared/payloads';
 
 const TemplateDownload: React.FC = () => {
   const h1Ref = getH1RefForTitle();
-  const { data: templateMetadata } = useSWR('template/metadata', {
```
Comment (JordanGuinn) [inline client/src/containers/TemplateDownload/TemplateDownload.tsx:11]: NICE - I didn't even notice this before 😎👏🏽

File: e2e-tests/tests/newIncompleteChild.js [hunk 3]
```diff
@@ -8,13 +8,19 @@ module.exports = {
...
+
     await login(browser);
-    // Navigate to roster
     await navigateToRoster(browser);
-    // Add child
-    await browser.click('xpath', `//*/a[contains(., 'Add a record')]`);
-    await browser.waitForElementVisible('body');
+
+    // In case the Alert would otherwise reset the scroll, wait and re-scroll
+    await browser.pause(3000);
```
Comment (JordanGuinn) [inline e2e-tests/tests/newIncompleteChild.js:16]: oh nightwatch 😭😭

**Review by JordanGuinn — APPROVED** [review]
This still looks good to me following the roster submission period updates 👌🏽

**Review by bamader — APPROVED** [review]
Agreed, still looks good. This is the hardest round of copy changes I've ever seen though 😭

**Comment by bamader** [comment]
## Validation Plan

- [ ] All references to July-Dec data are replaced with references to the new month deadline (including Data Requirements, Home, Roster
- [ ] Language is added to remind users to update data at least monthly
- [ ] Language explaining the data template is added
- [ ] Language explaining ECE Reporter is added to the home page
- [ ] Language explaining the file upload process is added to the wizard
- [ ] Language explaining the roster is added to the Roster page
- [ ] Roster submission:
  - [ ] Users cannot submit if there are any errors present in data
  - [ ] Users can submit if the data is error-free
  - [ ] After the roster is submitted, errors do not break the roster flow
- [ ] Users who have not submitted data see an alert on the home page reminding them to do so
  - [ ] This alert is persistent
- [ ] Users who have submitted data see no alert of any kind after the success modal
- [ ] Users who have submitted data have the Submit button disabled on the roster

**Comment by bamader** [comment]
Validations look good! I found only two super tiny things:
1. On the Data Requirements page, I think we should change the *Last Updated* date to today; the language asking to update data monthly is new from the perspective of the user, so to me that counts as a legitimate update
2. On the file upload page, there should be a space between `data collection template` and `to enter all...` (right now, templateto is all one word because of the link formatting)

**Comment by JordanGuinn** [comment]
Fixed and validated the remaining two points above, so I'd say this is READY

**Pull request merged**